namespaces/live.cloud-platform.service.justice.gov.uk/hale-platform-prod/resources/justiceinspectorates.tf

```diff
@@ -21,3 +21,83 @@ resource "kubernetes_secret" "justiceinspectorates_route53_zone" {
     zone_id = aws_route53_zone.justiceinspectorates_route53_zone.zone_id
   }
 }
+
+resource "aws_route53_record" "justiceinspectorates_route53_cname_record_main_www" {
+  zone_id = aws_route53_zone.justiceinspectorates_route53_zone.zone_id
+  name    = "www.justiceinspectorates.gov.uk"
+  type    = "CNAME"
+  ttl     = "3600"
+  records = ["inspectorates.live.bangdynamics.com"]
+}
+
+resource "aws_route53_record" "justiceinspectorates_route53_cname_record_main" {
```

Comment:
This needs to be changed. In Route35 you can't have a CNAME that matches the hostedzone name. Need to check what it should be.